game-core/src/main/java/games/strategy/triplea/delegate/AaInMoveUtil.java

```diff
@@ -116,13 +116,20 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
               // get rid of units already killed, so we don't target them twice
               validTargetedUnitsForThisRoll.removeAll(casualties);
               if (!validTargetedUnitsForThisRoll.isEmpty()) {
+                // Fly over AA currently doesn't take into account support
                 dice.set(
-                    DiceRoll.rollSbrOrFlyOverAa(
+                    DiceRoll.rollAa(
                         validTargetedUnitsForThisRoll,
                         currentPossibleAa,
                         AaInMoveUtil.this.bridge,
                         territory,
-                        BattleState.Side.DEFENSE));
+                        CombatValueBuilder.aaCombatValue()
+                            .enemyUnits(List.of())
```

Comment:
nit, you can make the API nicer here by defaulting enemyUnits to an empty list, then you won't need to specify it in the builder.

Reply:
I prefer that it is explicit.  Generally, the enemy list shouldn't be empty.  The only reason it is empty here is because support for AA guns during sbr or fly over isn't implemented (or maybe it isn't wanted).

Reply:
Okay.
side-note:  in that case take caution for a potential interface-segregation-principle violation here.